### PR TITLE
[12.0][MIG] partner_identification

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

File: partner_identification/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Partner Identification Numbers',
+    'category': 'Customer Relationship Management',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'contacts',
+    ],
+    'data': [
```
Comment (nikul-serpentcs) [inline partner_identification/__manifest__.py:17]: IMO data sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: partner_identification/__manifest__.py (hunk 1 above)
Comment (tarteo) [reply partner_identification/__manifest__.py:17]: Done

**Review by astirpe — CHANGES_REQUESTED** [review]
@tarteo Runbot fails for this trivial warning:

```
2018-10-24 11:59:36,149 151 WARNING openerp_test odoo.models: The model res.partner.id_number has no _description
2018-10-24 11:59:36,149 151 WARNING openerp_test odoo.models: The model res.partner.id_category has no _description
```

Could you fix it?

File: partner_identification/tests/test_res_partner.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError
+from .fake_models import ResPartner, setup_test_model, teardown_test_model
+
+
+class TestResPartner(common.SavepointCase):
+
+    at_install = False
```
Comment (lmignon) [inline partner_identification/tests/test_res_partner.py]: @tarteo PLZ remove `at_install` and `pot_install`

**Review by lmignon — CHANGES_REQUESTED** [review]

File: partner_identification/tests/test_res_partner.py (hunk 2 above)
Comment (tarteo) [reply partner_identification/tests/test_res_partner.py]: Thanks, removed them.

File: partner_identification/tests/test_res_partner.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        setup_test_model(cls.env, ResPartner)
+        bad_cat = cls.env['res.partner.id_category'].create({
+            'code': 'another_code',
+            'name': 'another_name',
+        })
+        cls.env['res.partner.id_number'].create({
```
Comment (nikul-serpentcs) [inline partner_identification/tests/test_res_partner.py:19]: @tarteo `cls.env['res.partner.id_number']` and `cls.env['res.partner.id_category']` use multipal time, IMO you can create global variable.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by lmignon — APPROVED** [review]
Thank you @tarteo LGTM (Code + Functional test)

I don't know why runbot is red... The module is installed and works as expected on runbot.

**Comment by astirpe** [comment]
Runbot is red because of https://github.com/OCA/partner-contact/pull/644#pullrequestreview-168725311

**Comment by lmignon** [comment]
@astirpe Thank you for the info...

**Review by astirpe — APPROVED** [review]

File: partner_identification/tests/test_res_partner.py (hunk 3 above)
Comment (tarteo) [reply partner_identification/tests/test_res_partner.py:19]: I rather keep the code not related to migration as is.

**Comment by mlaitinen** [comment]
This PR is stuck only because of codecov? Can I help somehow?

**Comment by pedrobaeza** [comment]
@mlaitinen in fact, it's stuck because the lack of a second review. If everything is OK, then approve it.

**Comment by pedrobaeza** [comment]
Oh, I see that there are 2 reviews already, so I merge.

**Pull request merged**